### PR TITLE
✨ PIC-1537 add minimal ReportManager entity

### PR DESCRIPTION
**Pull request by chris-faulkner-mojdt**
This is joined to the CourtReport entity. This is mapped, along with its active flag to the CourtReportMinimal response.

### CODE REVIEW

File: src/main/java/uk/gov/justice/digital/delius/transformers/CourtReportTransformer.java [hunk 1]
```diff
@@ -57,7 +60,19 @@ public static CourtReportMinimal courtReportMinimalOf(uk.gov.justice.digital.del
                                                                         .description(reportType.getDescription())
                                                                         .build())
                                                     .orElse(null))
+            .reportManagers(reportManagersOf(report.getReportManagers()))
             .build();
     }
 
+    private static List<ReportManager> reportManagersOf(final List<uk.gov.justice.digital.delius.jpa.standard.entity.ReportManager> reportManagers) {
+        return Optional.ofNullable(reportManagers).orElse(Collections.emptyList())
```
Comment (andymarke) [inline src/main/java/uk/gov/justice/digital/delius/transformers/CourtReportTransformer.java:68]: if you wanted you could drop the null check given JPA will face an empty list - but I suppose "belt and braces" :-)

**Review by andymarke — APPROVED** [review]
LGTM

File: src/main/java/uk/gov/justice/digital/delius/transformers/CourtReportTransformer.java (hunk 1 above)
Comment (chris-faulkner-mojdt) [reply src/main/java/uk/gov/justice/digital/delius/transformers/CourtReportTransformer.java]: Done

**Review by andymarke — APPROVED** [review]
LGTM

**Review by andymarke — APPROVED** [review]
LGTM

**Pull request merged**